packages/runtime/src/runtime/hooks/useNavigateModal.tsx

```diff
@@ -23,6 +29,16 @@ export const useNavigateModalScreen: EnsembleActionHook<
   const screenContext = useScreenContext();
   const customScope = useCustomScope();
   const storage = useEnsembleStorage();
+  const ensembleAction = useEnsembleAction(
+    isString(action) ? undefined : (action?.onModalDismiss as EnsembleAction),
```

Comment:
I don't think you need this - the `useEnsembleAction` hook will automatically return undefined.